src/openforms/submissions/views.py

```diff
@@ -192,6 +194,46 @@ def get_form_resume_url(self, submission: Submission) -> str:
         )
 
 
+class DownloadSubmissionReportView(PrivateMediaView):
+    """
+    Download the PDF report containing the submission data.
+
+    This URL requires a token which is tied to the submission from the session. The
+    token automatically expired after
+    ``settings.SUBMISSION_REPORT_URL_TOKEN_TIMEOUT_DAYS`` days.
+    """
+
+    queryset = SubmissionReport.objects.all()
+    pk_url_kwarg = "report_id"
+    file_field = "content"
+    raise_exception = True
+
+    object: SubmissionReport
+
+    def has_permission(self):
+        report: SubmissionReport = self.get_object()
+        token = self.kwargs["token"]
+        return submission_report_token_generator.check_token(report, token)
+
+    def get_sendfile_opts(self) -> dict:
+        path = Path(self.object.content.name)
+        opts = {
+            "attachment": True,
+            "attachment_filename": path.name,
```

Comment:
What is the reason to go through `Path(self.object.content.name).name` and not just `self.object.content.name`?

Reply:
the former only returns `[basename].[extension]`, the latter returns `some/subpath/from/upload/to/[basename].[extension]`